target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -118,7 +118,8 @@ case "$FIRMWARE" in
 		caldata_extract "art" 0x5000 0x844
 		ath10k_patch_mac $(macaddr_add $(cat /sys/class/net/eth0/address) -2)
 		;;
-	tplink,re450-v2)
+	tplink,re450-v2|\
+	tplink,re450-v3)
 		caldata_extract "art" 0x5000 0x844
 		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary info 0x8) +1)
```

Comment:
Have you checked with OEM firmware that the offset is actually +1?

Reply:
Yes.

Reply:
Yes.